### PR TITLE
Update README to include links to Xero Untax, Xero Items …

### PR DESCRIPTION
**Pull request by agileware-justin**
…extensions. Bonus minor README refactor.

Agileware Ref: CIVIXERO-44

### CODE REVIEW

**Comment by agileware-justin** [comment]
Seemed like a good idea, following Matt's recent PR https://github.com/eileenmcnaughton/nz.co.fuzion.civixero/pull/63

File: README.md [hunk 1]
```diff
@@ -1,32 +1,41 @@
...
+To modify the behaviour of the CiviXero extension various hooks are available. The additional CiviCRM extensions are available:
 
-See https://github.com/agileware/au.com.agileware.xerotweaks for an example usage of this.
+[Xero Tweaks](https://github.com/agileware/au.com.agileware.xerotweaks)
+- Removes the Contact ID from their Xero name.
+- Includes additional address lines in the contact address.
+- Removes the Contact's name from the Invoice Reference and Line items.
+
+[Xero Untax](https://github.com/agileware/au.com.agileware.xerountax)
+- Remove tax details from line items sent via CiviXero, so Xero can figure it out avoiding rounding issues.
```
Comment (eileenmcnaughton) [inline README.md]: Dang - I didn't think we were passing through tax details - did someone convince me to?

Comment (agileware-justin) [reply README.md]: That would have been our PR #18

Comment (eileenmcnaughton) [reply README.md]: ah...

**Pull request merged**